packages/transmute-dashboard/src/store/transmute/init.js

```diff
@@ -1,71 +0,0 @@
-
-// import { toast } from 'react-toastify';
```

Comment:
@eolszewski any chance we can fix this, instead of delete it?

It should never have been commented out, but I'm concerned we are hiding a feature here that needs to be implemented. If you don't want to fix it, please add a card for this feature to the backlog.

Reply:
@OR13 this is just cleaning up commented out code. This should not be conflated with the feature being delivered here. I do agree, though and have created an issue for this.

https://transmute.atlassian.net/browse/ENG-306